manager/container.go

```diff
@@ -633,6 +634,14 @@ func (cd *containerData) updateLoad(newLoad uint64) {
 	}
 }
 
+func (cd *containerData) updateLoadD(newLoad uint64) {
+	if cd.loadDAvg < 0 {
```

Comment:
How is this variable initialized with negative value?

Reply:
fixed.  loadDAvg initialized with -1 like loadAvg does.